tests/Makefile

```diff
@@ -67,11 +56,6 @@ iso-to-vm: build-box/sda.vdi deps
 	VBoxManage storageattach "test" --storagectl "sata controller" --port 1 --device 0 --type dvddrive --medium $(CLOUD_INIT_ISO)
 	VBoxManage startvm "test" --type headless
 
-clean_vm_from_iso:
```

Comment:
Why are you removing this? Its useful to have clean methods, especially for local deployment and testing

Reply:
Because it wasn't used anywhere, but I forgot the manual tests yes sorry... I will let it!